feat(xo-lite): calculate vdi chain physical usage for accurate storag… (#10116)
vatesfr/xen-orchestra: @xen-orchestra/lite/CHANGELOG.md

## **next**

> - Fix VDI storage used space calculation to include entire parent chain when snapshots present (PR [#10116](https://github.com/vatesfr/xen-orchestra/pull/10116))

Reviewer: ```suggestion
- [VDI] Fix used space not taking the whole parent chain into account when snapshots are present (PR [#10116](https://github.com/vatesfr/xen-orchestra/pull/10116))
```
